packages/manager/apps/vrack-services/src/App.tsx

```diff
@@ -28,7 +30,18 @@ export const App: React.FC = () => {
 
   return (
     <QueryClientProvider client={queryClient}>
-      <RouterProvider router={router} />
+      <UpdateDisplayNameContext.Provider
```

Comment:
Shouldn't we have a context for all type of messages ? Not only for display name messages ?

Reply:
https://github.com/ovh/manager/pull/11781/files#diff-ca8098eaca001628500d33defaea5d2f4a33e0a2a79767bf211f264cd593a6f9

Will be done in the next PR